kiso.libraries.yml

```
@@ -196,4 +196,4 @@ track-focus:
     base:
       css/base/elements/track-focus.css: {}
   js:
-    //cdn.jsdelivr.net/gh/ten1seven/what-input@v5.2.10/dist/what-input.min.js: { type: external, minified: true }
+    js/contrib/what-input/what-input.min.js: {}
```

@rutiolma Same question as for PR #104... To avoid bloading the **Kiso (基礎)** theme folder with a whole series of local JS libraries, wouldn't it be more appropriate to install "What Input?" inside the `/libraries` folder using the Openfed `composer.json` file, as we've already done for the "ARIA Accordion" library?

```suggestion
    /libraries/what-input/dist/what-input.min.js: { minified: true }
```

---

Indeed, this should be included in libraries. I'll add it to Openfed.

---

Thanks @rutiolma, be aware I made the same request for the "Splide" accessible slider/carousel TypeScript in the PR #104.